### ISSUE TITLE
Something went wrong: Invalid access token passed. on GET https://wporg.automattrix.com/_matrix/client/v3/account/whoami?.

### ISSUE THREAD

**Issue by aaronjorbin** [WordPress/matrix#24]
### Expected Behavior

When I visit https://make.wordpress.org/core/chat/ while logged in, i get the following error `Something went wrong: Invalid access token passed. on GET https://wporg.automattrix.com/_matrix/client/v3/account/whoami?.`

**Comment by akirk** [comment]
Could it be that you used the function of Element where you can log out other clients?

**Comment by aaronjorbin** [comment]
Nope. I saw this (and continue to see it) even before I downloaded element
